runtimes/pythonrt/requirements.txt

```
@@ -7,6 +7,7 @@ boto3 ~= 1.34
 google-api-python-client ~= 2.122
 google-auth-httplib2 ~= 0.2
 google-auth-oauthlib ~= 1.2
+google-generativeai ~= 0.7.1
```

This will pin the version to 0.7 but get the latest (and backwards compatible) minor version.

```suggestion
google-generativeai ~= 0.7
```